index.html

```diff
@@ -2988,62 +2989,86 @@ <h3>The <dfn>PropertyReadHandler</dfn> callback</h3>
 
     <section> <h3>The <dfn>emitPropertyChange()</dfn> method</h3>
       <div>
-        Takes the |name:string| argument, denoting a <a>Property</a> name.
+        Takes the |name:string| argument, denoting a <a>Property</a> name, and
+        optionally the |data:InteractionInput| argument.
         Triggers emitting a notification to all observers of the specified
-        <a>Property</a> with the data obtained from the observe handler or the
+        <a>Property</a> with the data provided by the |data| argument, or if not
+        provided, it is obtained by the observe handler or the by
```

Comment:
*Note*: the observe handler is not mentioned later on in the algorithm. Also, I think the observe handler now has another function: it is called when somebody observes the property.

Reply:
This is not related to this PR. Let's take a look at that problem in a different issue and PR.

Reply:
Ok, let's address this in a follow-up PR.